app/src/main/java/com/cornellappdev/android/eatery/ui/components/general/Filter.kt

```diff
@@ -2,69 +2,71 @@ package com.cornellappdev.android.eatery.ui.components.general
 
 import com.cornellappdev.android.eatery.data.models.Eatery
 
-enum class Filter(val text: String) {
-    UNDER_10("Under 10 min"),
-    NORTH("North"),
-    WEST("West"),
-    CENTRAL("Central"),
-    FAVORITES("Favorites"),
-    BRB("BRBs"),
-    CASH("Cash or credit"),
-    SWIPES("Meal swipes"),
-    SELECTED("Selected"),
-    TODAY("Today");
+sealed class Filter(open val text: String) {
+    sealed class CustomFilter(override val text: String) : Filter(text) {
+        data object Today : CustomFilter(text = "Today")
+        data object Selected : CustomFilter(text = "Selected")
+    }
```

Comment:
Hmm okay I actually really like the structure here! It requires some pattern matching with `when` when used but I think that's unavoidable given the fact that the other classes need different parameters.

Just one problem--is the `CustomFilter` kind of supposed to be a catch-all-else wild card? If so, is there any need necessarily for you to enumerate `Today` and `Selected`? Perhaps we can just make this a `data class CustomFilter(...)` that we can use to just make any custom filter. And then just an idea, maybe we can have it take in a custom `passesFilter: () -> Unit` anonymous function? So that it's flexible enough to run any filter?